### PR TITLE
Send 410 for expired ad feature facias

### PR DESCRIPTION
**Pull request by samdesborough**
Also use RenderOtherStatus in common IndexController so that appropriate
templates are rendered for non-200 results.

### CODE REVIEW

**Comment by samdesborough** [comment]
@stephanfowler @janua is the IndexControllerCommon change going to break anything? :)

File: common/app/controllers/IndexControllerCommon.scala [hunk 1]
```diff
@@ -6,6 +6,7 @@ import play.api.mvc._
 import services.Index
 import services.IndexPage
 import performance.MemcachedAction
+import views.support.RenderOtherStatus
```
Comment (ironsidevsquincy) [inline common/app/controllers/IndexControllerCommon.scala]: out of interest, why `Other`? why not just `RenderStatus`, or if they're all `4xx`s, `RenderErrorStatus`

**Comment by gklopper** [comment]
:-1: first talk to @JustinPinner about how we handle 410s. Even though this is the proper way to do it, this negatively affects search rankings

File: common/app/controllers/IndexControllerCommon.scala (hunk 1 above)
Comment (samdesborough) [inline common/app/controllers/IndexControllerCommon.scala]: good point, it only deals with errors so should probably be named as such.
there's a bit more to do on this, I'll look at renaming it then.

**Comment by JustinPinner** [comment]
@samdesborough and @kelvin-chappell dropped by about this - I explained that we return a 200 response but with the 410 page content elsewhere because GoogleBot doesn't deal with 410s properly.

**Comment by kelvin-chappell** [comment]
Actually this does end up with a 200 response.
See https://github.com/guardian/frontend/blob/master/common/app/views/support/package.scala#L178

**Comment by JustinPinner** [comment]
@kelvin-chappell yup - that's the bit we tweaked to return a 200 instead of a 410. If you wind up there all will be well.

**Comment by kelvin-chappell** [comment]
Then all will be well so :+1: for Monday unless more objections before then.

**Pull request merged**